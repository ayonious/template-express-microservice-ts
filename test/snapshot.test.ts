import sum from '../src/utils/utils';

describe('Test with snapshot', () => {
  it(`test1`, function () {
    const p = sum(10);
    expect(p).toMatchSnapshot();
  });
});
