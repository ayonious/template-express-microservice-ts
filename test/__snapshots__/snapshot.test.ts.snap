// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test with snapshot test1 1`] = `55`;
